common/environment/build-style/cargo.sh

```diff
@@ -4,6 +4,13 @@ if ! [[ "$pkgname" =~ ^cargo-auditable(-bootstrap)?$ ]]; then
 	hostmakedepends+=" cargo-auditable"
 fi
 
+if [ -n "$XBPS_SCCACHE" ] && [ "$pkgname" != "rust-sccache" ]; then
+	case "$XBPS_MACHINE" in
+		x86_64*|i686*|arm*|aarch64*) hostmakedepends+=" rust-sccache" ;;
+		*) ;;
+	esac
+fi
```

Comment:
I really don't like implicit build-style dependencies, especially with logic like this.

Reply:
I agree it's not the cleanest. But isn't it explicit since XBPS_SCCACHE is opt-in?

Reply:
With ppc support being dropped from void, don't need the arch checking logic - so this is as "clean" as it's going to be.